uqcsbot/scripts/yt.py

```diff
@@ -0,0 +1,54 @@
+import os
+from uqcsbot import bot, Command
+import argparse
+
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+YOUTUBE_API_KEY = 'AIzaSyCtxVMs6So6x2WL5WkDV4rm01hzddCCGH4'
```

Comment:
do you know if youtube api key == google api key? if so we can do `os.environ.get('GOOGLE_API_KEY')`

Reply:
nvm about google api key, turns out we have `YOUTUBE_API_KEY` :) drop that in as a replacement

Reply:
i.e. `YOUTUBE_API_KEY = os.environ.get('YOUTUBE_API_KEY')`

Reply:
The `YOUTUBE_API_KEY` or `GOOGLE_API_KEY` cause issues when dropping them in, is this because the keys are only available on the deployed bot?